packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/utils/cluster_manager.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../types.dart';
```

Comment:
I am not sure if the team has a rule against it but relative imports I tend to view as a code smell since it makes refactoring more fragile.

Reply:
Relative imports within `lib` are the encouraged practice: https://dart.dev/guides/libraries/create-library-packages#importing-library-files

Reply:
Ah thanks. This is my google3 bias coming through. Resolving.